dask/dataframe/utils.py

```diff
@@ -366,6 +370,25 @@ def make_meta_object(x, index=None):
     raise TypeError("Don't know how to create metadata from {0}".format(x))
 
 
+def make_meta_util(x, index=None, parent_meta=None):
+    import dask.dataframe as dd
+
+    if isinstance(x, (dd.core.Series, dd.core.DataFrame)):
+        return x._meta
+    if hasattr(x, "_meta"):
+        return x._meta
+
+    try:
+        return make_meta(x, index=index)
```

Comment:
Does this mean we still need to remove the `make_meta.register(object)` definition in dask-cudf before the original issue is resolved?  Not a problem if so, I am just trying to understand everything.

Reply:
Correct. We'll need `dask-cudf` side changes to accommodate this new dispatch - `make_meta_object`.